docs/docsite/rst/dev_guide/developing_program_flow_modules.rst

```diff
@@ -466,18 +462,71 @@ this. If a module has to use this on its own, it should instantiate an
 
 .. versionadded:: 2.1
 
+
 _ansible_version
 ^^^^^^^^^^^^^^^^
 
-This parameter passes the version of Ansible that runs the module. To access
-it, a module should instantiate an `AnsibleModule` and then retrieve it
-from :attr:`AnsibleModule.ansible_version`. This replaces
-:attr:`ansible.module_utils.basic.ANSIBLE_VERSION` from
-:ref:`module_replacer`.
+This argument passes the version of Ansible to the module. To access it, a module should instantiate the ``AnsibleModule`` method and then retrieve the version from :attr:`AnsibleModule.ansible_version`. This replaces :attr:`ansible.module_utils.basic.ANSIBLE_VERSION` from :ref:`module_replacer`.
 
 .. versionadded:: 2.1
 
 
+_ansible_module_name
+^^^^^^^^^^^^^^^^^^^^
+
+Type: ``str``
+
+This argument passes the information to modules about their name. For more details see, the environment variable :ref:`DEFAULT_MODULE_NAME`.
```

Comment:
`DEFAULT_MODULE_NAME` is a configuration option, not an environment variable, same for all the following references too. Config options might include environment variables, but also ini entries, variables, and cli options.